From c1285db5bd2c4047da144ae9f71026c5f8f846c2 Mon Sep 17 00:00:00 2001
From: Bruce Ashfield <bruce.ashfield@windriver.com>
Date: Thu, 19 Jun 2008 11:03:07 -0400
Subject: [PATCH] kbuild: exclude meta directory from distclean processing

The meta directory has some files (notably files of size 0) that are
removed by Kbuild's distclean processing. Add wrs to the exclude
list to prevent recursion into wrs.

Signed-off-by: Bruce Ashfield <bruce.ashfield@windriver.com>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 96407c1d6be1..fc1932aa092f 100644
--- a/Makefile
+++ b/Makefile
@@ -635,7 +635,7 @@ export KBUILD_AFLAGS_KERNEL KBUILD_CFLAGS_KERNEL KBUILD_RUSTFLAGS_KERNEL
 # Files to ignore in find ... statements
 
 export RCS_FIND_IGNORE := \( -name SCCS -o -name BitKeeper -o -name .svn -o    \
-			  -name CVS -o -name .pc -o -name .hg -o -name .git \) \
+			  -name CVS -o -name .pc -o -name .hg -o -name .git -o -name meta \) \
 			  -prune -o
 
 # ===========================================================================
-- 
2.39.2

